### PR TITLE
Feature/usage&cost

### PR DESCRIPTION
**Pull request by LisaQ04**

### CODE REVIEW

File: waterlog/src/Components/MonthlyUsage.js [hunk 1]
```diff
@@ -37,16 +35,15 @@ const MonthlyUsageComponent = (props) => {
         display: true,
         scaleLabel: {
           display: true,
-          labelString: 'Liters'
+          labelString: 'liters'
```
Comment (FredJEhlers) [inline waterlog/src/Components/MonthlyUsage.js:38]: If only these were in constants then you had to only change it once

File: waterlog/src/Components/Usage.js [hunk 2]
```diff
@@ -2,88 +2,126 @@ import React, { Component } from 'react';
...
   dailyUsage: state.dailyUsage.item,
   monthlyUsage: state.monthlyUsage.item,
-  seasonUsage: state.seasonUsage.items
+  seasonUsage: state.seasonUsage.items,
+  dailyCost: state.dailyCost.item,
+  monthlyCost: state.monthlyCost.item,
+  seasonalCost: state.seasonalCost.items
 })
-export default connect(mapStateToProps, { fetchUsageDaily,fetchUsageMonthly,fetchUsageSeasonally })(Usage);
+export default connect(mapStateToProps, { fetchUsageDaily, fetchUsageMonthly, fetchUsageSeasonally, fetchCostsDaily, fetchCostsMonthly, fetchCostsSeasonally })(Usage);
```
Comment (FredJEhlers) [inline waterlog/src/Components/Usage.js]: split over multiple lines

File: waterlog/src/Components/MonthlyUsage.js (hunk 1 above)
Comment (FredJEhlers) [reply waterlog/src/Components/MonthlyUsage.js:38]: I think, you should make use of constants here

Comment (LisaQ04) [reply waterlog/src/Components/MonthlyUsage.js:38]: Thanks for the input I will consider doing that.

File: waterlog/src/Components/Usage.js (hunk 2 above)
Comment (LisaQ04) [reply waterlog/src/Components/Usage.js]: done.

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**